.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,9 +18,10 @@ Fixes #
 - [ ] If adding new functionality, add an example to docstrings or tutorials.
 - [ ] Use underscores (not hyphens) in names of Python files and directories.
 
-**Slash Commands**
+**Bot Commands**
```

Comment:
Changes in this file can be reverted.

Reply:
Done at 386ad8e9eba7f4a8ebfac4a13805db91d6b89c46